### PR TITLE
better output channel names

### PR DESCRIPTION
**Pull request by nmschulte**
for #98

### CODE REVIEW

File: firmware/console/binary/output_channels.txt [hunk 1]
```diff
@@ -13,28 +13,28 @@ bit sd_present
...
-bit launchTriggered;
-bit isTps2Error;
-bit isIdleClosedLoop;
-bit injectorFault;
-bit ignitionFault;
+bit sd_msd;SD is MSD
+bit isFan2On;Radiator Fan2
+bit acrActive;Holley ACR Active
+bit toothLogReady;Tooth Logger Ready
+bit isTpsError;TPS Error
```
Comment (mck1117) [inline firmware/console/binary/output_channels.txt:25]: How do we feel about something like `Error: TPS` so that these alphabetize better?

Comment (nmschulte) [reply firmware/console/binary/output_channels.txt:25]: Sounds good to me.  Same for "Warning: ...", "Ready: ...", "Active: ..." ("Triggered: ..." -- change to "Active"?), "Fault: ..." (change to "Error"?)?

**Comment by nmschulte** [comment]
force push fixes commit message typo ...

File: firmware/console/binary/output_channels.txt (hunk 1 above)
Comment (mck1117) [reply firmware/console/binary/output_channels.txt]: The other stuff doesn't seem to have many similar ones for different systems, let's just start with error flags for sensors and such

File: firmware/console/binary/output_channels.txt [hunk 2]
```diff
@@ -13,28 +13,28 @@ bit sd_present
...
-bit isWarnNow;
-bit isPedalError;
-bit launchTriggered;
-bit isTps2Error;
-bit isIdleClosedLoop;
-bit injectorFault;
-bit ignitionFault;
+bit sd_msd;SD is MSD
+bit isFan2On;Radiator Fan2
+bit acrActive;Holley ACR Active
```
Comment (mck1117) [inline firmware/console/binary/output_channels.txt:23]: Harley?

Comment (nmschulte) [reply firmware/console/binary/output_channels.txt]: Of course.

**Pull request merged**